management/src/test/java/org/kaazing/gateway/management/jmx/TlsTestUtil.java

```diff
@@ -0,0 +1,43 @@
+package org.kaazing.gateway.management.jmx;
+
+import java.io.FileInputStream;
+import java.security.KeyStore;
+
+public class TlsTestUtil {
+    static char[] password = "ab987c".toCharArray();
+    static String keyStoreFileLocation = "/Users/David/Desktop/kaazing-websocket-gateway-4.0.9/conf/keystore.db";
```

Comment:
How do we resolve`keyStoreFileLocation` absolute path when these tests are run in different environments?